test/integration/pipeline/multisig_tx_pipeline_test.cpp

```diff
@@ -105,14 +78,39 @@ class MstPipelineTest : public AcceptanceFixture {
    */
   auto makeGetPendingTxsQuery(const std::string &creator,
                               const crypto::Keypair &key) {
-    return shared_model::proto::QueryBuilder()
-        .createdTime(iroha::time::now())
-        .creatorAccountId(creator)
-        .queryCounter(1)
-        .getPendingTransactions()
-        .build()
-        .signAndAddSignature(key)
-        .finish();
+    return complete(baseQry(creator).getPendingTransactions(), key);
+  }
+
+  /**
+   * Query validation lambda - check that empty transactions response returned
+   * @param response - query response
+   */
+  static void no_txs_check(const proto::QueryResponse &response) {
+    ASSERT_NO_THROW({
+      const auto &pending_txs_resp =
+          boost::get<interface::TransactionsResponse>(response.get());
```

Comment:
`interface::TransactionsResponse` -> `const interface::TransactionsResponse &` 
Same in other methods
You can make a local shortcut which will accept `T` and do `boost::get<const T &>`